extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/reactive/runtime/ReactiveMongoOperations.java

```diff
@@ -551,6 +551,12 @@ private static Document sortToDocument(Sort sort) {
         return collection.deleteMany(new Document()).map(deleteResult -> deleteResult.getDeletedCount());
     }
 
+    public static Uni<Boolean> deleteById(Class<?> entityClass, Object id) {
+        ReactiveMongoCollection<?> collection = mongoCollection(entityClass);
+        Document query = new Document().append(ID, id);
+        return collection.deleteOne(query).map(results -> results.getDeletedCount() == 1);
```

Comment:
See? This one seems optimal, so let's not be unfavourable to ORM ;)

Reply:
Yes because the ID field name is always `_id` in MongoDB so it's easy to implement ;)